samples/snippets/csharp/concepts/methods/byref106.cs

```diff
@@ -1,22 +1,20 @@
 ﻿// <Snippet106>
-using System;
-
-public class ByRefExample
+public static class ByRefExample
 {
-   public static void Main()
-   {
-      int value = 20;
-      Console.WriteLine("In Main, value = {0}", value);
-      ModifyValue(ref value);
-      Console.WriteLine("Back in Main, value = {0}", value);
-   }
+    public static void Main()
+    {
+        var value = 20;
+        Console.WriteLine("In Main, value = {0}", value);
+        ModifyValue(ref value);
+        Console.WriteLine("Back in Main, value = {0}", value);
+    }
 
-   static void ModifyValue(ref int i)
-   {
-      i = 30;
-      Console.WriteLine("In ModifyValue, parameter value = {0}", i);
-      return;
-   }
+    static void ModifyValue(ref int i)
```

Comment:
add access modifier:

```suggestion
    private static void ModifyValue(ref int i)
```